websqrl/next.config.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     config.resolve.fallback = {
       // the require for this module is wrapped in a try/catch, so it can fail without causing issues
       "sqrl-test-utils": false,
+      "re2": false,
```

Comment:
@meyer this doesn't seem to work and I have no idea why.

I'm getting this error:
```
error - ../node_modules/re2/re2.js:3:0
Module not found: Can't resolve './build/Release/re2'

Import trace for requested module:
../packages/sqrl-text-functions/lib/index.js
./workers/compile.worker.ts
./components/StreamPage.tsx
./pages/twitter.tsx
```

Reply:
re2 is a native dependency so webpack can’t import it unless there’s a wasm alternative it can use (i found [re2-wasm](https://github.com/google/re2-wasm) but it looks a bit… unmaintained). The `builtinRegExp` config option bypasses the re2 import but re2 is still getting imported since the `builtinRegExp` config option might be `false`. The `config.builtinRegExp` check should be combined with something that webpack can understand at build time if we want the re2 import to be elided. Something like so (caution: untested code):

```js
  let createRegExp: CreateRegExp;
  if (process.env.IS_WEBPACK || config.builtinRegExp) {
    createRegExp = (source, flags) => new RegExp(source, flags);
  } else {
    const RE2 = require("re2"); // if we’re cool with promises, this can be `import('re2')` instead
    createRegExp = (source, flags) => new RE2(source, flags);
  }
```

Using `webpack.DefinePlugin` we can set `process.env.IS_WEBPACK` to `false` at build time so the re2 branch of the if statement is marked as unused.